docs/install/mac_pkg.md

```diff
@@ -33,7 +33,6 @@ cmake .. -DCMAKE_INSTALL_PREFIX=$NRN_INSTALL \
   -DNRN_PYTHON_DYNAMIC="$pythons" \
   -DIV_ENABLE_X11_DYNAMIC=ON \
   -DNRN_ENABLE_CORENEURON=OFF \
-  -DNRN_ENABLE_INTERNAL_READLINE=ON \
```

Comment:
I wonder if we need to have the same (new) handling like for the wheels: build `readline` and `ncurses` with `MACOSX_DEPLOYMENT_TARGET=10.9`. If yes, then it may be enough to add a new script for that with build instructions from the Dockerfile.